test/Microsoft.TemplateEngine.Edge.UnitTests/LocalizationTests.cs

```diff
@@ -142,43 +145,33 @@ public void TestLocalizedSymbolDescription(string locale, string expectedDescrip
             Assert.Equal(expectedManualInstructions, effects.CreationResult.PostActions[postActionIndex].ManualInstructions);
         }
 
-        private ITemplateEngineHost LoadHostWithLocalizationTemplates(string locale, out SettingsLoader settingsLoaderOut, out TemplateInfo localizationTemplate)
+        class TemplatesFactory : ITemplatePackageProviderFactory
```

Comment:
should default factory be used here? (if needed at all)

Reply:
I think its fine to have different factories for different tests, because it allows to execute tests in parallel, notice that Factory is kind of singleton... So it can't be shared... Provider implementation is shared...